build-configs/integreat/assets/app-logo-g.svg

```diff
@@ -0,0 +1,19 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="59" height="59" viewBox="0 0 59 59">
```

Comment:
Maybe you could use a more speaking name for this svg :)

Reply:
What do you suggest? Shall we use the use-case (app-loading) or a descriptive name how it looks  (app-integreat-g)?

I used app-logo-g because it is a g :)

Reply:
Ahh okay. Couldn't make any sense of g ^^. Maybe integreat-logo-inverted would make sense as it is basically the logo with the colors switched :)

Reply:
done